topics/aws/README.md

```diff
@@ -1851,7 +1851,7 @@ False. It's disabled by default
 <details>
 <summary>True or False? In regards to cross zone load balancing, AWS charges you for inter AZ data in network load balancer but no in application load balancer</summary><br><b>
 
-False. It charges for inter AZ data in network load balancer, but not in application load balancer
+True. It charges for inter AZ data in network load balancer, but not in application load balancer
```

Comment:
Can you please explain the change in the details?

Reply:
**Question:** True or False? In regards to cross zone load balancing, AWS charges you for inter AZ data in network load balancer but no in application load balancer.
- Ans should be **True**

**Ref:** 
1. [ALB](https://aws.amazon.com/elasticloadbalancing/faqs/#:~:text=Q%3A%C2%A0Am%20I%20charged,of%20regional%20data%20transfer)
2. [NLB](https://aws.amazon.com/elasticloadbalancing/faqs/#:~:text=Q%3A%20Am%20I%20charged%20for%20regional%20AWS%20data%2Dtransfer,of%20the%20Amazon%20EC2%20On%2DDemand%20Pricing%20page.)